dts/bindings/pinctrl/realtek,rts5912-pinctrl.yaml

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2024 Realtek Semiconductor Corporation, SIBG-SD7
+#
+
+description: |
+     This binding gives a base representation of the pins configuration
+
+compatible: "realtek,rts5912-pinctrl"
+
+include: [base.yaml, pinctrl-device.yaml, pincfg-node.yaml]
+
+properties:
+  reg:
+    required: true
+
+child-binding:
+  description: |
+    This binding gives a base representation of the pins configuration
+
+  include:
+    - name: pincfg-node.yaml
+
```

Comment:
missing docs, list of allowed properties?

Reply:
We have added properties below into allowed list.
https://github.com/zephyrproject-rtos/zephyr/pull/75267#discussion_r1867227937